### ISSUE TITLE
Keyword arguments are not supported

### ISSUE THREAD

**Issue by matt-royal** [benmoss/quacky#2]
Hi Ben,

I used Quacky to test a method in ruby 2.0 that takes keyword arguments, and noticed that it did not enforce the keyword argument names. In other words, this:

```
def foo(a: nil, b: nil); end
```

quacks like this, according to quacky:

```
def foo(completely_different_argument_name: nil, b: nil); end
```

Thanks,
Matt

**Comment by benmoss** [comment]
Yeah, I'm not sure kwargs were ever considered (if they even existed at the time). Same answer applies as the other issue, I'm afraid.
